os/fs/driver/mtd/smart.c

```diff
@@ -521,6 +525,28 @@ static const struct block_operations g_bops = {
  ****************************************************************************/
 
 #ifdef CONFIG_DEBUG_FS
+#ifndef NXFUSE_HOST_BUILD
+static int smart_dumpext(FAR struct smart_struct_s *dev)
```

Comment:
smart level doesn't know storage is internal or external.
Please make common name.

Reply:
Updated to your previously used name
"smart_dumpall"